src/esp/gfx/PbrIBLHelper.cpp

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Meta Platforms, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include "PbrIBLHelper.h"
+#include <Magnum/GL/Mesh.h>
+#include <Magnum/GL/Texture.h>
+#include <Magnum/GL/TextureFormat.h>
+#include <Magnum/Image.h>
+#include <Magnum/ImageView.h>
+#include <Magnum/Magnum.h>
+#include <Magnum/Math/Angle.h>
+#include <Magnum/MeshTools/Compile.h>
+#include <Magnum/MeshTools/Copy.h>
+#include <Magnum/MeshTools/FlipNormals.h>
+#include <Magnum/Primitives/Cube.h>
+#include <Magnum/Trade/AbstractImporter.h>
+#include <Magnum/Trade/ImageData.h>
+#include <Magnum/Trade/MeshData.h>
+
+#include "CubeMapCamera.h"
+
+namespace Mn = Magnum;
+namespace Cr = Corrade;
+
+namespace esp {
+namespace gfx {
+
+namespace {
+// TODO: move to PbrShaderAttributes
+constexpr unsigned int environmentMapSize = 1024;
+constexpr unsigned int prefilteredMapSize = 1024;
+constexpr unsigned int irradianceMapSize = 128;
+
+Mn::Matrix4 buildDfltPerspectiveMatrix() {
+  using Mn::Math::Literals::operator""_degf;
+  return Mn::Matrix4::perspectiveProjection(
+      90.0_degf,  // horizontal field of view angle
+      1.0f,       // aspect ratio (width/height)
+      0.01f,      // z-near plane
+      1000.0f);   // z-far plane
+}  // buildDfltPerspectiveMatrix()
+
+}  // namespace
+
+PbrIBLHelper::PbrIBLHelper(
+    ShaderManager& shaderManager,
+    const std::shared_ptr<Mn::GL::Texture2D>& brdfLUT,
+    const std::shared_ptr<Mn::GL::Texture2D>& envMapTexture)
+    : shaderManager_(shaderManager), brdfLUT_(brdfLUT) {
+  // using the brdflut from here:
+  // https://github.com/SaschaWillems/Vulkan-glTF-PBR/blob/master/screenshots/tex_brdflut.png
+
+  // loadBrdfLookUpTable(blutImageData);
```

Comment:
reason to keep this around?

Reply:
Nope, oversight on my part. Thanks for finding it.